cms/djangoapps/contentstore/views/course.py

```diff
@@ -283,6 +293,41 @@ def course_rerun_handler(request, course_key_string):
             })
 
 
+@login_required
+@ensure_csrf_cookie
+@require_http_methods(["GET"])
+def course_index_handler(request, course_key_string):
+    """
+    The restful handler for course indexing.
+    GET
+        json: return status of indexing task
+    """
+    # Only global staff (PMs) are able to rerun courses during the soft launch
+    if not GlobalStaff().has_user(request.user):
+        raise PermissionDenied()
+    course_key = CourseKey.from_string(course_key_string)
+    with modulestore().bulk_operations(course_key):
+        if request.method == 'GET':
+            error = reindex_course_and_check_access(course_key, request.user, depth=3)
+            if error:
+                return HttpResponse(
+                    json.dumps({"status": error}),
+                    content_type='application/json'
+                )
+            return HttpResponse(
+                json.dumps({"status": 'success'}),
```

Comment:
You should not need to say `"status": "success"`. The HTTP status code should indicate success or failure of the response.